examples/text-layer/text-layer/text-layer.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable max-len */
```

Comment:
Yep. Actually, it would be nice to have it as a core layer so that for other text-related layer dev (e.g., tagmap) we do not need to duplicate the code.